newIDE/app/src/ResourcesList/ResourceUtils.js

```diff
@@ -80,7 +80,26 @@ export const copyAllToProjectFolder = (
         return resourcePath;
       }
 
-      const resourceBasename = path.basename(resourcePath);
+      let resourceBasename = path.basename(resourcePath),
+        splitAt = resourceBasename.lastIndexOf('.'),
+        fileName = resourceBasename.substring(0, splitAt),
+        fileExtension = resourceBasename.substring(splitAt),
+        fileNumber = 1;
+
+      const AllResourcesNames = project
+        .getResourcesManager()
+        .getAllResourceNames();
+
+      for (let itr = 0; itr < AllResourcesNames.size(); itr++) {
+        if (fileName + fileExtension === AllResourcesNames.at(itr)) {
+          fileName = resourceBasename.substring(0, splitAt) + fileNumber;
+          fileNumber++;
+          itr = 0;
+        }
+      }
+
+      resourceBasename = fileName + fileExtension;
+
       const resourceNewPath = path.join(projectPath, resourceBasename);
```

Comment:
No need to modify resourceBasename, you can do directly like this:
```suggestion
      const resourceNewPath = path.join(projectPath, fileName + fileExtension);
```
This makes the intention more clear and let you put `resourceBasename` as const to show it is a constant (the base name)